lib/output/mqtt.go

```diff
@@ -2,6 +2,7 @@ package output
 
 import (
 	"github.com/Jeffail/benthos/v3/internal/docs"
+	mqttconf "github.com/Jeffail/benthos/v3/internal/mqttconf"
```

Comment:
`mqttconf` before shouldn't be required here since it is the package name

Reply:
Indeed. Strange that the linter didn't catch this one. Fixed.